examples/puppetmaster/test.sh

```diff
@@ -261,158 +251,29 @@ conjur::ssl_certificate: |
 $ssl_certificate
   " > $hiera_config_file
 
-  revoke_cert_for "$hostname"
-
-  set -x
-  docker run --rm -t \
-    --net $NETNAME \
-    --add-host "conjur.cyberark.com:$DOCKER_GATEWAY_IP" \
-    --hostname "$hostname" \
-    "$agent_image" \
-      agent --verbose \
-            --onetime \
-            --no-daemonize \
-            --summarize \
-            --certname "$hostname"
-  set +x
-
-  rm -rf "$hiera_config_file"
-}
-
-converge_node_hiera_hft() {
-  local agent_image="$1"
-  local node_name="hiera-hft-node"
-  local hostname="${node_name}_$(openssl rand -hex 3)"
-
-  local login="host/$node_name"
-  local hf_token=$(get_hf_token)
-  echo "HF token for $node_name: $hf_token"
-
-  local hiera_config_file="./code/data/nodes/$hostname.yaml"
-
-  local ssl_certificate="$(cat https_config/ca.crt | sed 's/^/  /')"
-  echo "---
-lookup_options:
-  '^conjur::host_factory_token':
-    convert_to: 'Sensitive'
-
-conjur::account: 'cucumber'
-conjur::appliance_url: 'https://conjur.cyberark.com:$(conjur_host_port)'
-conjur::authn_login: 'host/$node_name'
-conjur::host_factory_token: '$hf_token'
-conjur::ssl_certificate: |
-$ssl_certificate
-  " > $hiera_config_file
-
-  revoke_cert_for "$hostname"
-
-  set -x
-  docker run --rm -t \
-    --net $NETNAME \
-    --add-host "conjur.cyberark.com:$DOCKER_GATEWAY_IP" \
-    --hostname "$hostname" \
-    "$agent_image" \
-      agent --verbose \
-            --onetime \
-            --no-daemonize \
-            --summarize \
-            --certname "$hostname"
-  set +x
-
-  rm -rf "$hiera_config_file"
-}
-
-converge_node_manifest_apikey() {
```

Comment:
I take it that we'll eventually need to add smoketests back in for the different ways of providing Conjur identity (other than the Hiera method)?

Reply:
Nope - there's only 2 ways to do things now and they are both implemented in this PR:
- On-agent credentials
- In-manifest credentials

The second item when using hiera covers both hiera use case and in-manifest credentials case at the same time